charts/pega/templates/_pega-deployment.tpl

```diff
@@ -187,8 +187,10 @@ spec:
         - name: {{ template "pegaVolumeCredentials" }}
           mountPath: "/opt/pega/secrets"
         #mount custom certificates
+{{ if .root.Values.global.certificates }}
```

Comment:
minor nitpick: "#mount custom certificates" comment should be within the condition. Currently it will render the comment regardless of if you have custom certs in your values.yaml